src/Umbraco.Cms.Api.Management/Controllers/Element/Tree/ChildrenElementTreeController.cs

```diff
@@ -0,0 +1,27 @@
+using Asp.Versioning;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.ViewModels.Tree;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Element.Tree;
+
+[ApiVersion("1.0")]
+public class ChildrenElementTreeController : ElementTreeControllerBase
+{
+    public ChildrenElementTreeController(IEntityService entityService, IUmbracoMapper umbracoMapper)
+        : base(entityService, umbracoMapper)
+    {
+    }
+
+    [HttpGet("children")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedViewModel<ElementTreeItemResponseModel>), StatusCodes.Status200OK)]
+    public async Task<ActionResult<PagedViewModel<ElementTreeItemResponseModel>>> Children(CancellationToken cancellationToken, Guid parentId, int skip = 0, int take = 100, bool foldersOnly = false)
```

Comment:
Consider if we need the `Guid? dataTypeId = null` parameter (as we have on documents  and media for children, root and siblings endpoints).  Might be necessary if we consider a picker for elements and we define start folders for users for elements.